authorization/src/main/java/com/terabite/authorization/repository/LoginNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.terabite.authorization.repository;
+
+public class LoginNotFoundException extends RuntimeException {
```

Comment:
Consider extending from NoResourceFoundException
See docs: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/reactive/resource/NoResourceFoundException.html
Also investigate if the controller will pass the error message from this extension as part of a json response if you allow the exception to propagate up the stack, past the controller, so that Spring handles it. If it does, then it's a nice replacement for `ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Could not find user with token: "+token); ` and similar bad response entities